### PR TITLE
Build xtest with "-DDEBUG -O0 -g3" when "make DEBUG=1"

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: host/xtest/Makefile [hunk 1]
```diff
@@ -133,6 +133,10 @@ CFLAGS += -Wall -Wcast-align -Werror \
 	  -Wno-missing-field-initializers -Wno-format-zero-length
 endif
 
+ifeq ($(DEBUG),1)
+CFLAGS += -DDEBUG -O0 -g3
```
Comment (jenswi-linaro) [inline host/xtest/Makefile]: What about always using `-g3` regardless of `DEBUG`?

Comment (jforissier) [inline host/xtest/Makefile]: You mean just add `CFLAGS += -g3`? I'm OK with that.

Comment (jenswi-linaro) [inline host/xtest/Makefile]: Yes

**Comment by jforissier** [comment]
Updated to always build with `-g3` (Android too!).

**Comment by jenswi-linaro** [comment]
The `DEBUG` flag was quite useful too, but doing only the `-g3` right now is OK.

**Comment by jforissier** [comment]
OK let's close this PR and open a new one for `-g3` only.

**Pull request closed without merging**